services/catarse/spec/controllers/admin/balance_transactions_controller_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::BalanceTransactionsController, type: :controller do
+  subject { response }
+  let(:admin) { create(:user, admin: true) }
+  let(:current_user) { admin }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(current_user)
+    allow(controller).to receive(:authenticate_user!).and_return(true)
+  end
+
+  describe "POST transfer_balance" do
+    let(:sender) { create(:user) }
+    let(:receiver) { create(:user) }
+    let(:amount) { rand(1...1000) }
+
+    context "When user has admin roles" do
```

Comment:
**Cosmético:**
Usar letra minúscula no início dos describe, context e it pra manter a padronização.